### ISSUE TITLE
touch device: open fountain detail window is very hard

### ISSUE THREAD

**Issue by ralfhauser** [mmmatthew/water-fountains.org#183]
tapping with your finger often doesn't open window

probably the touch area needs to be increased on small-/touch- screens

(I thought to have reported this approx a week ago, but don't find it anymore :(  )

Complementary measure is #182

**Comment by ralfhauser** [comment]
Testing again, I notice that it improved since last week - good

**Comment by ralfhauser** [comment]
When trying to open
https://beta.water-fountains.org/zurich?lang=de&queryType=byId&database=wikidata&idval=Q55170336 on my cell, got
![musikInstitut](https://user-images.githubusercontent.com/1681677/56095022-dd48e680-5ed8-11e9-9d57-788ed1e306ea.jpg)

**Comment by mmmatthew** [comment]
Is the "hard to open detail" still an issue?

**Comment by ralfhauser** [comment]
Yes - just observed a novice user on a A4 size Apple iPad Pro and opening the details was so hard that it exceeded a comparably patient attention span of the user

**Comment by mmmatthew** [comment]
This is possibly a duplicate of #38. Can we confirm that it just occurs for ios devices?
